kubernetes/kubernetes_service.go

```diff
@@ -225,6 +225,20 @@ func (in *IstioClient) GetDeployment(namespace, deploymentName string) (*v1beta1
 	return in.k8s.AppsV1beta1().Deployments(namespace).Get(deploymentName, emptyGetOptions)
 }
 
+// GetDeploymentSelector returns the delector of a deployment.
```

Comment:
typo here in "delector" -> "selector"

Reply:
Also, if the string returned has some kind of structure (i.e. a comma separated list of) it should be put it in the comment.
As I need to check the FormatLabels() function to discover if the string is plain or if it is defined in some way.

Reply:
it's a sepatrated ','
This is the magic method 
```go
// String returns all labels listed as a human readable string.
// Conveniently, exactly the format that ParseSelector takes.
func (ls Set) String() string {
	selector := make([]string, 0, len(ls))
	for key, value := range ls {
		selector = append(selector, key+"="+value)
	}
	// Sort for determinism.
	sort.StringSlice(selector).Sort()
	return strings.Join(selector, ",")
}
```

Reply:
Comment added